src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
 
-[Learn more about Dev Portal &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/dev-portal/)
+{{site.base_gateway}} allows you to store sensitive data fields, such as consumer secrets, in an encrypted format within the database.
+This provides encryption-at-rest security controls in a {{site.base_gateway}} cluster.
+
+[Set up keyring and data encryption &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/db-encryption/)
 
 ## Audit logging
 
-{{site.base_gateway}} provides a granular logging facility on its Admin API. This
-allows cluster administrators to keep detailed track of changes made to the
-cluster configuration throughout its lifetime, aiding in compliance efforts and
+{{site.base_gateway}} provides granular logging of the Admin API. You can keep detailed track of changes made to the
+cluster configuration throughout its lifetime, for compliance efforts and to
 providing valuable data points during forensic investigations. Generated audit
 log trails are [workspace](/gateway/{{page.kong_version}}/admin-api/workspaces/reference) and [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference)-aware,
 providing {{site.base_gateway}} operators a deep and wide look into changes happening within
 the cluster.
 
 [Get started with audit logging &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/audit-log/)
 
-## Keyring and data encryption
-
-{{site.base_gateway}} allows you to store sensitive data fields, such as consumer secrets, in an encrypted format within the database. This provides encryption-at-rest security controls in a {{site.base_gateway}} cluster.
-
-This functionality provides transparent, symmetric encryption of sensitive data fields at rest. Transparency refers to the fact that, when enabled, encryption and decryption of data is done by {{site.base_gateway}} immediately before writing, or immediately after reading from the database. Responses generated by the Admin API containing sensitive fields continue to show data as plaintext, and runtime elements of {{site.base_gateway}} (such as plugins) that require access to sensitive fields do so transparently, without requiring additional configuration.
-
-[Set up keyring and data encryption &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/db-encryption/)
-
-## Roles-based access control (RBAC)
+## FIPS support
 
-You can configure {{site.base_gateway}} users, roles, and permissions with role-based access control (RBAC), either through the [Admin API](/gateway/{{page.kong_version}}/admin-api/rbac/reference/) or with [Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/rbac). RBAC lets you manage access to resources in {{site.base_gateway}}.
+Kong Enterprise is self-managed and features a FIPS 140-2 compliant gateway, making it ideal for highly regulated industries with strict compliance and security considerations. 
```

Comment:
```suggestion
Kong Enterprise features a self-managed FIPS 140-2 gateway package, making it ideal for highly regulated industries with strict compliance and security considerations. 
```
We can't say that it's FIPS compliant. It hasn't been audited/approved for that.